README.md

```diff
@@ -36,6 +37,21 @@ $ make
 ### Using the ISO
 - The ISO can be used with [minishift](https://github.com/minishift/minishift) with the help of `--iso-url` i.e. `--iso-url=file:///$PATHTOISO`.
 
+<a name="tests"></a>
+## Tests
+
+Tests are written as Python scripts under `tests` directory and run through [Avocado](avocado-framework.readthedocs.io) framework. Ensure you have _Avocado_ installed by following [Avocado instllation guide](http://avocado-framework.readthedocs.io/en/44.0/GetStartedGuide.html#installing-avocado) before running any tests.
```

Comment:
@hferentschik , It can be install via [`pip`](http://avocado-framework.readthedocs.io/en/44.0/GetStartedGuide.html#installing-from-standard-python-tools).

@amitkrout has installed in Mac OS https://paste.fedoraproject.org/525918/48421257.